package.json

```diff
@@ -44,6 +44,7 @@
     "danger": "danger ci",
     "prettier": "prettier --config .prettierrc --write \"**/*.js\"",
     "create-component": "node scripts/npm/create-component.js",
+    "submodules:init": "git submodule update --init --recursive",
```

Comment:
So in hind-sight, @matthewdavidson and @tiagoengel, I think it's best we leave the scripts as they are. However, I am going to propose adding this script to make it easy for us to undo accidentally commited submodule changes.